### PR TITLE
 Implemented week 03 homework without optional

### PR DESCRIPTION
**Pull request by Heema5657**
completed yuppie

### CODE REVIEW

File: javascript/javascript1/week3/homeworkweek03.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
+const nameToRemove = 'Ahmad';
+
+// Write some code here
+names.splice(1,1);
```
Comment (sowmya1408) [inline javascript/javascript1/week3/homeworkweek03.js]: Good job!!
Could you try to write a function which can remove any name from the array.

File: javascript/javascript1/week3/homeworkweek03.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+// Code done
+
+console.log(names); 
+// Exercise 2 
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  function notThisFunctionName(travelInformation){
+          const a = travelInformation.destinationDistance / travelInformation.speed;
```
Comment (sowmya1408) [inline javascript/javascript1/week3/homeworkweek03.js]: Variable naming&  also function naming conventions , hour/minute - is a general naming, be very specific.
ex: travelInHour etc...

File: javascript/javascript1/week3/homeworkweek03.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    {
+      title: 'The Wire',
+      days: 2,
+      hours: 12,
+      minutes: 0,
+    }
+  ]
+  function logOutSeriesText() {
+    // write code here
+    let noOfYearsInHours = 80 * 365 * 24;
```
Comment (sowmya1408) [inline javascript/javascript1/week3/homeworkweek03.js]: const keywords can be used in all this variables.

File: javascript/javascript1/week3/homeworkweek03.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+        totalOfTime += totalPercentage;
+        console.log(`${titleOfSeries} took ${totalPercentage.toFixed(3)}% of my life`);
+        
+    }
+    console.log(`In total this is ${totalOfTime.toFixed(3)}% of my life`)
+  }
+  logOutSeriesText();
+
+  //Exercise 4
+  let notes = [];
```
Comment (sowmya1408) [inline javascript/javascript1/week3/homeworkweek03.js]: perfect 💯

**Review by sowmya1408 — CHANGES_REQUESTED** [review]
Great job!!
Refer to below for variable/function naming convention.
https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md

File: javascript/javascript1/week3/homeworkweek03.js (hunk 4 above)
Comment (Heema5657) [reply javascript/javascript1/week3/homeworkweek03.js]: Thanks a lot.

File: javascript/javascript1/week3/homeworkweek03.js (hunk 1 above)
Comment (Heema5657) [reply javascript/javascript1/week3/homeworkweek03.js]: do you mean random ?

Comment (sowmya1408) [reply javascript/javascript1/week3/homeworkweek03.js]: Yes, now you can remove only the name 'Ahmad', write a function which can remove any name within the array.

**Review by sowmya1408 — APPROVED** [review]
Perfetc 💯

**Pull request merged**